xml/gl.xml

```diff
@@ -10339,7 +10339,8 @@ typedef unsigned int GLhandleARB;
         <enum value="0x8F67" name="GL_MAX_SHADER_PIXEL_LOCAL_STORAGE_SIZE_EXT"/>
             <unused start="0x8F68" vendor="ARM"/>
         <enum value="0x8F69" name="GL_TEXTURE_ASTC_DECODE_PRECISION_EXT"/>
-            <unused start="0x8F6A" end="0x8F6F" vendor="ARM"/>
+        <enum value="0x8F6A" name="GL_TEXTURE_UNNORMALIZED_COORDINATES_ARM"/>
```

Comment:
```suggestion
        <enum value="0x8F6A" name="GL_TEXTURE_UNNORMALIZED_COORDINATES_ARM" group="SamplerParameterF,SamplerParameterI,GetTextureParameter,TextureParameterName"/>
```

Reply:
I couldn't get the Commit suggestion to work, but pushed the same change manually. Sanity-checked that it didn't change headers (other than generating new dates). Resolving.